test/data/upgrade8to9/upgrade_verify.sh

```diff
@@ -2,7 +2,7 @@
 
 set -euxo pipefail
 
-source /usr/libexec/tests/osbuild-composer/shared_lib.sh
```

Comment:
I'm probably missing some more context, but isn't the osbuild-composer-tests RPM installed on the upgraded system? If not, then this makes sense, if it is, then I'm not sure why is this change needed. Could you please elaborate? Thanks.

Reply:
Just the `osbuild-composer` package from the latest nightly is installed on the test VM. See https://github.com/osbuild/osbuild-composer/blob/main/test/data/upgrade8to9/upgrade_prepare.sh#L11

So I copy the script over to the test VM so it can be sourced. The change is visible in this commit.